plugins/module_utils/postgres.py

```diff
@@ -82,7 +82,10 @@ def ensure_required_libs(module):
     if not HAS_PSYCOPG2:
         module.fail_json(msg=missing_required_lib('psycopg2'))
 
-    if module.params.get('ca_cert') and LooseVersion(psycopg2.__version__) < LooseVersion('2.4.3'):
+    elif PSYCOPG_VERSION < LooseVersion("2.8"):
+        module.warn("psycopg should be at least 2.8 to support all modules functionality")
+
+    if module.params.get('ca_cert') and PSYCOPG_VERSION < LooseVersion('2.4.3'):
         module.fail_json(msg='psycopg2 must be at least 2.4.3 in order to use the ca_cert parameter')
```

Comment:
can't this go away since we now require 2.8 and 2.8 > 2.4.3?

Reply:
> I didn't want to be too harsh. The modules will print out a warning when psycopg < 2.8, but we don't need to actively remove the code where it isn't in a way.

agreed, imo it's better not to touch if not necessary:)